doc/prebuild_tools.md

```diff
@@ -0,0 +1,15 @@
+# Prebuild tools
+
+The distribution of a native add-on is equally important than its implementation.
```

Comment:
The distribution of a native add-on is just as important as it's implementation.